#!/usr/bin/env python
PACKAGE = "rov_slidder"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Depth_SetPoint", int_t, 0, "D",    50, 0,   100)

gen.add("Kp", double_t, 0, "A",    5, 0,   10)
gen.add("Kd", double_t, 0, "B",    5, 0,   10)
gen.add("Ki", double_t, 0, "C",    5, 0,   10)

gen.add("PWM1", int_t, 0, "E",    1500, 1100,   1900)
gen.add("PWM2", int_t, 0, "F",    1500, 1100,   1900)
gen.add("PWM3", int_t, 0, "G",    1500, 1100,   1900)
gen.add("PWM4", int_t, 0, "H",    1500, 1100,   1900)
gen.add("PWM5", int_t, 0, "I",    1500, 1100,   1900)
gen.add("PWM6", int_t, 0, "J",    1500, 1100,   1900)

exit(gen.generate(PACKAGE, "server", "Tutorials"))

